app/extensions/brave/locales/en-US/autofill.properties

```diff
@@ -1,18 +1,21 @@
-autofillTitle=Autofill Settings
-addresses=Addresses
-creditCards=Credit Cards
 addAddress=Add Address
 addCreditCard=Add Credit Card
-name=Name
+addresses=Addresses
+autofillData=Autofill data
+autofillTitle=Autofill Settings
+city=City
+country=Country
```

Comment:
@srirambv another example:

````
addresses_city=...
addresses_country=...

...

creditCards_edit=...
creditCards_expirationDate=...
````

The hierarchy has not been explicitly specified with a certain rule which we should follow when adding/editing the localization strings. By creating that rule it will also be easy to remove duplicates.

Reply:
@luixxiul definitely something we'd want proposed in its own issue